indent/verilog_systemverilog.vim

```diff
@@ -277,8 +278,8 @@ function GetVerilog_SystemVerilogIndent()
       \ last_line !~ '^\s*\<\(sequence\|clocking\|interface\|covergroup\)\>'  &&
       \ last_line !~ '^\s*\<\(property\|program\)\>' &&
       \ last_line !~ '^\s*\()*\s*;\|)\+\)\s*' . vlog_comment . '*$' && (
-        \ last_line =~
-        \ '\<\(`\@<!if\|`\@<!else\|for\|while\|case\%[[zx]]\|always\|initial\|do\|foreach\|randcase\|final\)\>' ||
+        \ last_line =~ '\<\(`\@<!if\|`\@<!else\|for\|while\|case\%[[zx]]\||initial\|do\|foreach\|randcase\|final\)\>' ||
```

Comment:
Please try to modify only what is necessary, for the reasons explained above.
I'm being picky, I know :)